praw/models/reddit/submission.py

```diff
@@ -143,9 +143,18 @@ def _fetch(self):
     def _info_path(self):
         return API_PATH['submission'].format(id=self.id)
 
-    def hide(self):
-        """Hide Submission."""
-        self._reddit.post(API_PATH['hide'], data={'id': self.fullname})
+    def hide(self, other_submissions=None):
+        """Hide Submission.
+
+        :param other_subreddits: When given hide provided list of subs
+
+        """
+        if other_submissions:
+            submissions = ','.join([str(self.fullname)] +
```

Comment:
The logic here doesn't quite look right. And since `fullname` is already a string, this can probably be simplified to:

```python
submissions = ','.join([self.fullname] + [x.fullname for x in other_submissions])
```